### ISSUE TITLE
Voxel grid scale and points not updated after transform

### ISSUE THREAD

**Issue by Eric-Vin** [mikedh/trimesh#2033]
When I create a simple voxel grid and then scale it, the `scale` and `points` property are not updated accordingly. I believe this is a bug since the underlying code definitely reaches into the transform to get the information for both. Perhaps a caching issue? 

Here's a code snippet to reproduce:

```
encoding = [
    [[0, 0, 0], [0, 1, 0], [0, 0, 0]],
    [[0, 1, 1], [0, 1, 0], [1, 1, 0]],
    [[0, 0, 0], [0, 1, 0], [0, 0, 0]],
]
vg = trimesh.voxel.VoxelGrid(numpy.asarray(encoding))

scale = numpy.asarray([12,23,24])
s_matrix = numpy.eye(4)
s_matrix[:3, :3] *= scale

print(vg.scale)
print(vg.points)
vg.apply_transform(s_matrix)
print(vg.scale)
print(vg.points)
```

Which on my machine prints:
```
[1. 1. 1.]
[[0. 1. 1.]
 [1. 0. 1.]
 [1. 0. 2.]
 [1. 1. 1.]
 [1. 2. 0.]
 [1. 2. 1.]
 [2. 1. 1.]]
[1. 1. 1.]
[[0. 1. 1.]
 [1. 0. 1.]
 [1. 0. 2.]
 [1. 1. 1.]
 [1. 2. 0.]
 [1. 2. 1.]
 [2. 1. 1.]]

```

**Comment by mikedh** [comment]
Thanks for the report! I pushed a fix to my release-candidate branch `drop/py2` which I'm going to try to do a `--pre` release on in the next week or two. The caching was indeed screwy in voxel transforms, and I added your example as a test. 

The voxel stuff doesn't get a lot of maintenance, and any PR's/fixes/refactors super welcome if you're using it! I had the idle though of trying to use it for the non-embree ray option for broad phase ray-triangle queries (i.e. `trimesh.ray.ray_triangle.ray_candidates`).
